.github/workflows/docker-publish-full.yml

```diff
@@ -21,87 +22,128 @@ concurrency:
   cancel-in-progress: true
   group: ${{ github.workflow }}-${{ github.ref }}
 
+# The name for the produced image at ghcr.io.
 env:
-  IMAGE_NAME: wetterdienst-full
+  GHCR_IMAGE_NAME: earthobservations/wetterdienst-full
 
 jobs:
-  # Run tests.
-  # See also https://docs.docker.com/docker-hub/builds/automated-testing/
-  test:
+  build_and_test:
     runs-on: ubuntu-latest
 
     steps:
-      - name: Acquire sources
+      -
+        name: Acquire sources
         uses: actions/checkout@v3
-
-      - name: Setup Python
+      -
+        name: Setup Python
         uses: actions/setup-python@v4
         with:
           python-version: "3.10"
-
-      - name: Setup Poetry
+      -
+        name: Setup Poetry
         uses: snok/install-poetry@v1
-
-      - name: Build wheel package
+      -
+        name: Build wheel package
         run: poetry build --format=wheel
-
-      - name: Run tests
+      -
+        name: Upload wheel package
+        uses: actions/upload-artifact@v2
+        with:
+          name: ${{ runner.os }}-wheel-${{ github.sha }}
+          path: dist/wetterdienst-*.whl
+          retention-days: 7
+      -
+        name: Run tests
         run: |
           if [[ -f .github/release/full.test.yml ]]; then
             docker-compose --file .github/release/full.test.yml build
             docker-compose --file .github/release/full.test.yml run sut
           fi
 
-  # Push image to GitHub Packages.
-  # See also https://docs.docker.com/docker-hub/builds/
-  push:
-    # Ensure test job passes before pushing image.
-    needs: test
-
+  docker:
+    needs: build_and_test
     runs-on: ubuntu-latest
-    if: github.event_name == 'push'
-
     steps:
-      - name: Acquire sources
+      -
+        name: Acquire sources
         uses: actions/checkout@v3
-
-      - name: Setup Python
-        uses: actions/setup-python@v4
         with:
-          python-version: "3.10"
-
-      - name: Setup Poetry
-        uses: snok/install-poetry@v1
-
-      - name: Build wheel package
-        run: poetry build --format=wheel
-
-      - name: Build Docker image
-        run: docker build . --file .github/release/full/Dockerfile --tag $IMAGE_NAME
-
-      - name: Log into GitHub Container Registry
-        run: echo "${{ secrets.CR_PAT }}" | docker login https://ghcr.io -u ${{ github.actor }} --password-stdin
```

Comment:
Apparently, this was used beforehand to log in to GHCR.

Reply:
I added you both as admins for the packages (if that was a blocker before).

Reply:
Thanks!

I just tried to use `secrets.CR_PAT` instead of `github.token` but it is empty for me because I probably don't have the rights to access the secret.

Reply:
Still happens.

```
ERROR: failed to push ghcr.io/earthobservations/wetterdienst-standard:pr-872: unexpected status: 403 Forbidden
```

-- https://github.com/earthobservations/wetterdienst/actions/runs/4193093528/jobs/7269564063#step:11:435

Reply:
I've now permitted GHA **administrative** permissions to the packages.

![image](https://user-images.githubusercontent.com/453543/219343606-682bf4c8-4bcb-44fb-bd00-b9ac90a56b81.png)

Reply:
For another perspective, I've just created GH-874, with the same commits.